tests/providers/apache/cassandra/hooks/test_cassandra.py

```diff
@@ -72,20 +72,17 @@ def setUp(self):
         hook.shutdown_cluster()
 
     def test_get_conn(self):
-        with mock.patch.object(Cluster, "connect") as mock_connect, mock.patch(
-            "socket.getaddrinfo", return_value=[]
```

Comment:
The previous mocking didn't work anymore, but this was to some extend, testing the internals of Cassandra.

I have replaced it with just ensuring we call `Cluster() correctly here.

Reply:
This likely no longer works because cython "caches"/looks up the built in getaddrinfo at compile time, so the mocking has no effect.